src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figure {
+    private int firstSide;
+    private int secondSide;
+
+    public RightTriangle(int firstSide, int secondSide, String color) {
+        super(color);
+        this.firstSide = firstSide;
+        this.secondSide = secondSide;
+    }
+
+    @Override
+    public double getArea() {
+        return firstSide * secondSide / 2;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Figure: rectangle, area: " + String.format("%.1f", getArea())
+                + " sq. units, first side: " + firstSide + " units, secondSide: " + secondSide
+                + " units, color: " + getColor());
```

Comment:
Use the `name()` method for getting String representation of enum constants instead of `getColor()` if `color` is an enum type. If `color` is a String, ignore this comment.